interface/src/avatar/MyAvatar.cpp

```diff
@@ -1533,7 +1536,19 @@ void MyAvatar::storeAvatarEntityDataPayload(const QUuid& entityID, const QByteAr
 
 void MyAvatar::clearAvatarEntity(const QUuid& entityID, bool requiresRemovalFromTree) {
     // NOTE: the requiresRemovalFromTree argument is unused
+
+    if (!DependencyManager::get<NodeList>()->getThisNodeCanRezAvatarEntities()) {
+        qCDebug(interfaceapp) << "Ignoring clearAvatarEntity() because don't have canRezAvatarEntities permission on domain";
```

Comment:
what if someone ACTUALLY tries to delete an avatar entity in this case?  or if its lifetime expires or a script tries to kill it?

Reply:
The avatar entities aren't in the tree so can't be operated on in the tree.

Reply:
by “tree” do you mean entity tree?  avatar entities are in the entity tree for clients, just not on the avatar mixer.

(I think)

Reply:
I mean the client entity tree. When avatar entities cannot be displayed, they're removed from the client entity tree in `MyAvatar::handleCanRezAvatarEntitiesChanged()`. Because they're not in the tree the cannot be deleted.